### PR TITLE
Mod 5 Project

### LINKED ISSUE #365: Mod 5 Project

### PR DESCRIPTION
**Pull request by memcmahon**
In progress... now ready for review

### CODE REVIEW

File: module5/projects/DemoGuide.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: Demo Guide
+---
+
+A well developmed project domo is a great opportunity to showcase your hard work and to practice talking about a project in front of a group of people. Below is a guide on how to present a project.
```
Comment (zoefarrell) [inline module5/projects/DemoGuide.md]: demo not domo

File: module5/projects/DemoGuide.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+
+## Guidelines
+
+1. __Important:__ Start the presentation by actually demoing your app. It is recommended that you skip demoing the log in/log out functionality. You can mention it in the demo, but start your demo already logged in. This will help save you time to talk about the more interesting features. Plan to live demo your application, but have a recording ready just in case there are unexpected issues. The recording is a nice backup.
```
Comment (zoefarrell) [inline module5/projects/DemoGuide.md:11]: I'm thinking we are adamant that they don't have any login/logout auth stuff as part of their MVP. Thoughts on including this in the MVP section above? And including something here about skipping demoing log in / log out if that is part of your app's functionality?

File: module5/projects/Capstone.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+This will be an individual project, and the product is largely up to you!  We will give you some guidelines to make sure you are demonstrating your learning, but the topic and much of the functionality of your project is all your own.
+
+The project will be split into 4 Phases:
+* Ideation
+* Inception
+* Development
+* Presentation/Documentation
+
+### Project Requirements
+Your application must include the following:
```
Comment (zoefarrell) [inline module5/projects/Capstone.md:17]: This feels like a lot for two weeks, but I do think we want them all pushing for this.

I'm just trying to think if we want to have softer wording than "requirements". Maybe "Project Expectations". Just trying to think of what we do if a student project doesn't meet these "requirements" like they don't have full crud working.

Comment (memcmahon) [reply module5/projects/Capstone.md:17]: Good call - I like project expectations... I'll also include something about prioritizing the API consumption.

File: module5/projects/DemoGuide.md (hunk 2 above)
Comment (memcmahon) [reply module5/projects/DemoGuide.md:11]: Yep, I like that callout in the MVP!

**Pull request merged**